tests/integrationv2/common.py

```diff
@@ -1,4 +1,25 @@
 import subprocess
+import string
+
+
+def data_bytes(n_bytes):
+    """
+    Generate bytes to send over the TLS connection.
+    These bytes purposefully fall outside of the ascii range
+    to prevent triggering "connected commands" present in
+    some SSL clients.
+    """
+    byte_array = [0] * n_bytes
```

Comment:
We purposely avoid certain characters because they could activate connected commands. If we allowed totally random characters, we would fail 3/256 runs due to key update commands being intercepted (and not sent to the peer) and the session closing early.